src/granule_to_sns.py

```diff
@@ -38,6 +38,12 @@ def granule_to_sns(event: dict, _) -> dict:
         client.publish(
             TopicArn=sns_topic_arn,
             Message=message,
+            MessageAttributes={
+                "collection": {
+                    "DataType": "String",
+                    "StringValue": granule["dataType"],
```

Comment:
I wonder if we want to add a way to get more fine grained filtering as well for the combined collections. For instance, for anything in the anc/aux collection this will be `NISAR_ANC_AUX`.

Reply:
I think that is a good idea. The ticket's acceptance criteria only asked for collections at the moment. I am happy to add it in the future. It just isn't related to ticket PR-5870.

Reply:
Might even do it Monday morning after I work on my slides.